src/main/panel-view.js

```diff
@@ -3,13 +3,13 @@
 var vscode = require('vscode');
 import { StatusBarAlignment, window } from 'vscode';
 
-import Term from './terminal';
-import ApiWrapper from '../main/api-wrapper.js';
+const Term = require('./terminal');const Term = require('./terminal')const Term = require('./terminal')
+const ApiWrapper = require('../main/api-wrapper.js');const ApiWrapper = require('../main/api-wrapper.js')
```

Comment:
Check this one!

Reply:
Thanks for catching this. The search/replace bugged up and made a bunch of these double statements. I thought I had found them all. I'll clean them up.